Sources/TuistCore/Utils/XcodeBuildLocator.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import TSCBasic
+import TuistGraph
+import TuistSupport
+
+public protocol XcodeBuildLocating {
```

Comment:
Nitpick: What about renaming this one to `XcodeProjectBuildDirectoryLocating`. It's a bit longer but more explicit. I think it'll rule out any future confusion about what the utility is for.

Reply:
If you agree with renaming I'd do it with all the definitions in this file, including the error.

Reply:
Yeah considered this but kept the name that @natanrolnik had used but I like the explicit name more.